uber/templates/fields/attendee.html

```diff
@@ -1151,14 +1152,20 @@ <h4>{{ tier.name }}</h4>
 {% endset %}
 
 
-{% set accessibility_interest %}
-{% set read_only = accessibility_interest_ro or page_ro %}
+{% set requested_accessibility_services %}
```

Comment:
Renamed this so it matched the column, making it easier to find. Also added code that hides this for admins who do not have accessibility access; unfortunately, we still _have_ to include the value of the column itself to avoid the value being reset.